authentication/handler_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
+// +build linux
```

Comment:
is this really required? Where would it fail (windows or mac)?

Reply:
Looks like it is happy, resolving this